core/src/main/resources/javascript/locales/de.json

```diff
@@ -1,5 +1,5 @@
 {
   "externalProvider": {
-    "label": "Benutzer und Gruppen Anbieter"
+    "label": "Benutzer- und Gruppen-Provider"
```

Comment:
Are we ok with that extra "-" after Benutzer ?

Reply:
Yes, because we do a listing of words. We just shorten it - instead of "Benutzer-Provider and Gruppen-Provider" we reduce.

Reply:
sounds good